WooCommerce/WooCommerceUITests/Tests/LoginTests.swift

```diff
@@ -31,16 +31,13 @@ class LoginTests: XCTestCase {
         XCTAssert(WelcomeScreen.isLoaded())
     }
 
-    func testLoginOptionsArePresent() {
-        //verify tapping "Log in" will show options to enter a WordPress.com email, log in with Google, or enter your site address.
-        WelcomeScreen()
+    func testEnterEmailAndSendLinkOrEnterPassword() {
+        //enter email and click next to verify presence of send link and password option
+        _ = WelcomeScreen()
```

Comment:
I do not know how to resolve the "result of call is unused" error other than to make it discardable, but if a result is discardable, why am I checking for it? I looked at the other written test for guidance but I don't see how mine was different.